arbnode/batch_poster.go

```diff
@@ -183,7 +183,6 @@ type BatchPosterConfig struct {
 	EspressoTxnsPollingInterval  time.Duration `koanf:"espresso-txns-polling-interval"`
 	EspressoSwitchDelayThreshold uint64        `koanf:"espresso-switch-delay-threshold"`
 	EspressoMaxTransactionSize   uint64        `koanf:"espresso-max-transaction-size"`
-	EspressoTEEVerifierAddress   string        `koanf:"espresso-tee-verifier-address"`
```

Comment:
This seems unnecesary for the batch poster.

We can use `lightClientAddress` and `espressoClient` to check if we are using espresso mode